src/objects/GetStartedChecklistSection__mdt.object

```diff
@@ -13,6 +13,32 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Page_Type__c</fullName>
+        <description>The type of page (Admin or End User) that the options in this GetStarted Section will be rendered in.</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The type of page (Admin or End User) that the options in this GetStarted Section will be rendered in.</inlineHelpText>
```

Comment:
@fparodi-salesforce Please change this to "The type of Get Started page (Admin or End User) where this section will appear."